Makefile

```diff
@@ -97,3 +152,104 @@ vet: ## Apply go vet to all go files
 .PHONY: help
 help:
 	@grep -E '^[a-zA-Z/0-9_-]+:.*?## .*$$' $(MAKEFILE_LIST) | sort | awk 'BEGIN {FS = ":.*?## "}; {printf "\033[36m%-30s\033[0m %s\n", $$1, $$2}'
+
+clean:
+	rm -rf $(OUTPUT_DIR)
+	oc delete mutatingwebhookconfigurations vpa-webhook-config || true
+	oc delete ns openshift-vertical-pod-autoscaler || true
+	oc delete crd verticalpodautoscalercheckpoints.autoscaling.k8s.io verticalpodautoscalercontrollers.autoscaling.openshift.io verticalpodautoscalers.autoscaling.k8s.io || true
```

Comment:
I would rather these oc commands be part of a separate target. Typically `clean:` is just for removing build artifacts and I think people would be surprised to see it trying to talk to their cluster.

Maybe they could be in a target called `clean-deploy:` or something like that?

Also, maybe it would be good to use a `$(KUBECTL)` variable like you do for other targets, instead of `oc`